### ISSUE TITLE
Add ansible-lint to Travis build

### ISSUE THREAD

**Issue by siddharthist** [CiscoCloud/mantl#1255]
This is currently blocked by https://github.com/willthames/ansible-lint/issues/113. Whether or not it's fixed in master can be tested with:
```
git clone "https://github.com/CiscoCloud/mantl"
git clone "https://github.com/willthames/ansible-lint"
cd mantl 
find . -name "*.yml" -exec ../ansible-lint/bin/ansible-lint {} +
```
This currently produces a traceback.

**Comment by siddharthist** [comment]
Closing as duplicate of #946
